vmware_exporter/vmware_exporter.py

```diff
@@ -167,6 +172,19 @@ def collect(self, target=None, section='default'):
                         'VMWare Host Memory Max availability in Mbytes',
                         labels=['host_name']),
                 }
+        collect_subsystems = self._collect_subsystems(section, metric_list.keys())
+
+
+        metrics = {}
+        for s in collect_subsystems:
+            metrics.update(metric_list[s])
+
+
+        collect_subsystems = self._collect_subsystems(section, metric_list.keys())
```

Comment:
I think there is a mistake here (duplicates)

Reply:
It thing mistake after merge.
In my fork another 183 line.